department-of-veterans-affairs/va.gov-cms
# VACMS-12930: Retool vagovclaro to use webpack, remove gulp

## Description

Closes #12930

## Testing done
Tested locally

## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As engineer
1. Check the build logs
   - [ ] Validate that the theme compiles correctly. 
   - [ ] Verify that nvm install does note result in `nvm: command not found`

As design/a11y
1. Check the site for design and a11y issues
   - [ ] Validate that there are no new issues. 


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
